x-pack/plugins/fleet/public/index.ts

```diff
@@ -64,3 +64,6 @@ export { PackagePolicyEditorDatastreamMappings } from './applications/fleet/sect
 export type { PackagePolicyEditorDatastreamMappingsProps } from './applications/fleet/sections/agent_policy/create_package_policy_page/components/datastream_mappings';
 
 export type { DynamicPagePathValues } from './constants';
+
+export { PackageListGrid } from './applications/integrations/sections/epm/components/package_list_grid';
```

Comment:
The changes look good, though it's concerning that the bundle size increases by this much with the exports. I'm not really sure how to reduce this.
```
fleet	157.6KB	785.1KB	⚠️ +627.5KB	❌ 614.6KB over [limit](https://github.com/elastic/kibana/blob/46a562abe69472411b960175f8bba8ea63c9014c/packages/kbn-optimizer/limits.yml)
```

Reply:
@elastic/kibana-operations - What is the recommendation for resolving bundle overages for plugins that need to export code for sharing with other plugins? This seems like a legitimate reason for the bundle size to increase, and I don't know what we could necessarily do as the maintainer of the newly-shareable code to deal with this. 

One option that comes to mind is moving all of Fleet's "shareable" code to a new plugin and therefore a new bundle, but that seems extreme to unblock this PR. Curious if you have recommendations here.

Reply:
How many plugins do we expect to import this?  Is moving it to a package in support of async imports an option?